recipes-bsp/tools/setup-nv-boot-control/uefi_common.func.in

```diff
@@ -130,7 +129,7 @@ gen_compat_spec() {
 # boardspec should be piped into this function
 needs_mmc_hack() {
     IFS=- read boardid fab boardsku boardrev fuselevel chiprev
-    if [ "$boardid" = "2888" -a "$fab" = "400" ]; then
+    if [ "$boardid" = "2888" ]; then
```

Comment:
This should probably be changed to something like
```
if [ "$boardid" = "2888" ] && [ "$boardsku" = "0004" -o "$fab" = "400" ]; then
```
which should align with the change in the gen_compat function.  The AGXi does have SPI flash, unlike the normal AGX.

Reply:
Ok, I wasn't sure about agxi. This doesn't handle the 0005 / 64 gb though, so maybe I'll invert the logic to exclude agxi

Reply:
Ok, I updated it so that boardsku 0008 (agxi) is excluded for 2888, assuming all other agx boards will need this going forward regardless of fab